modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -166,14 +166,12 @@ Add a list of certificates that were revoked by certificate authorities before t
 
 == Edit a Secret Group
 
-When you edit a secret group, other users are locked out while you edit the necessary security objects. If a user doesn't end an editing session by either finishing or canceling it, the *Edit* button is disabled for other users. If you click *Cancel Edit*, the open editing session is closed, and the *Edit* button becomes available, even if the session was started by a different user. 
+Edit a secret group to add secret types such as Keystore, Truststore, and Certificates.
 
+. In the *Secret Groups* list view, select the secret group you want to edit, and click *Edit*. 
+. Make your changes as required, then click *Finish.*
 
-. In the *Secret Group* list view, click *Edit* for the secret group that you want to update. 
-. Make your changes and click *Finish*. +
-The secret group is updated with the changes, and the secrets manager serves the updated secrets to all redeployed applications after this point. 
-. Click *Cancel Edit* to close an open editing session. +
-Use caution when you cancel another user's editing session, because all the changes made by the other user will be lost.
+The secret group is updated with the changes, and the secrets manager provides the updated secrets to all redeployed applications after this point. 
```

Comment:
This sentence is not correct anymore. The save secrets for each type needs an update like that seen below: 

"When each secret is saved, the updated secret is available for use by the next application deployment or the next access of the secret."

Reply:
After clicking *Finish*, Secrets Manager saves your changes. The updated secret group is immediately available the next time the secret group is accessed or when deploying an application.